### PR TITLE
[ADD] udes_sale_stock: Add flag in picking view for privacy wrapping

### PR DESCRIPTION
**Pull request by nocturnalastro**
User-stoy 1946
Signed-off-by: Michele Costa <michele.costa@unipart.io>

### CODE REVIEW

File: addons/udes_sale_stock/models/stock_move.py [hunk 1]
```diff
@@ -23,3 +28,9 @@ def _action_cancel(self):
             self.mapped('sale_line_id.order_id').check_delivered()
 
         return result
+
+    def _compute_privacy_wrapped(self):
+        privacy = self.env.ref('udes_stock.privacy_wrapping')
+        for mv in self:
+            mv.u_is_privacy_wrapped = privacy in mv.mapped(
```
Comment (ws-mps) [inline addons/udes_sale_stock/models/stock_move.py]: One move only links to one sale_line_id, so no need for mapped.

File: addons/udes_sale_stock/models/stock_move.py [hunk 2]
```diff
@@ -9,6 +9,11 @@ class StockMove(models.Model):
 
     _inherit = 'stock.move'
 
+    u_is_privacy_wrapped = fields.Boolean(
+        string='Is privacy wrapped',
+        compute='_compute_privacy_wrapped',
```
Comment (ws-mps) [inline addons/udes_sale_stock/models/stock_move.py]: Another option is to use a related field, with readonly=True, related='sale_line_id.privacy_wrapping' ? Although it is not going to be a boolean field.

File: addons/udes_sale_stock/views/stock_picking_views.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_form_stock_picking" model="ir.ui.view">
+        <field name="name">udes_sale_picking_form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='move_lines']/tree/field[@name='product_id']" position="after">
+                <field name="u_is_privacy_wrapped" />
```
Comment (ws-mps) [inline addons/udes_sale_stock/views/stock_picking_views.xml]: readonly ?

File: addons/udes_sale_stock/models/stock_move.py (hunk 2 above)
Comment (nocturnalastro) [reply addons/udes_sale_stock/models/stock_move.py]: I like that better, I'll ask what would be preferred tomorrow. :)

**Pull request closed without merging**